.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "editor.formatOnSave": true,
+  // "editor.formatOnSave": true,
```

Comment:
Please remove. In the future, remember to review your status (`git status`) before committing files in order to avoid this. It's very good practice and will get you out of trouble